core/domain/draft_upgrade_services.py

```diff
@@ -88,6 +88,37 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v32_dict_to_v33_dict(cls, draft_change_list):
+        """Converts draft change list from state version 32 to 33. State
+        version 33 adds showChoicesInShuffledOrder boolean variable to the
+        MultipleChoiceInput interaction.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        for i, change in enumerate(draft_change_list):
+            if (change.cmd == exp_domain.CMD_EDIT_STATE_PROPERTY and
+                    change.property_name ==
+                    exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS):
+                if (len(change.new_value) == 1 and
+                        'choices' in change.new_value.keys()):
```

Comment:
Have you verified explicitly that this does not describe the customization args for any other interaction?

You should also have a test that includes customization args for a different interaction and verify that those are not affected. (In general, the need for this sort of test should come to mind automatically when you write the code, so please think proactively in the future about how to test robustly.)

Reply:
I have indeed verified that this does not describe the cust args for any other interactions. As you can see I have included the the ItemSelectionInput cust args(second one) which is similar to multipleChoiceInput and tested that it is not affected.

Reply:
Ah, ok, thanks! Sorry, that was my bad. I misread the second case!